config/index.js

```diff
@@ -465,6 +465,13 @@ var conf = convict({
       ],
       env: 'SIGNIN_CONFIRMATION_FORCE_EMAIL_REGEX'
     }
+  },
+  securityHistory: {
+    enabled: {
+      doc: 'enable security history',
+      default: true,
+      env: 'SECURITY_HISTORY_ENABLED'
+    }
```

Comment:
It's unfortunate that this feature-flag can't pref off the updated `verifyTokens_2` query in auth-db-server, meaning that we'll still try to verify security events even if we're not recording them.  Probably not much we can do about that though.